index.html

```diff
@@ -0,0 +1,460 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <h1 id="logo">Aziz Mejri</h1>
+      <button type="button" class="menu-bars">
+        <i class="fas fa-bars"></i>
+      </button>
+      <div class="dropdown-menu hide">
+        <div class="mobile-menu">
+          <button class="close">&times;</button>
+        </div>
+        <ul class="mobile-menu-options">
+          <li class="dropdown-option"><a href="#projects">Portfolio</a></li>
+          <li class="dropdown-option"><a href="#aboutme">About</a></li>
+          <li class="dropdown-option"><a href="#contact">Contact</a></li>
+        </ul>
+        <div class="dropdown-bottom">
+          <div class="dropdown-divline"></div>
```

Comment:
Just as the previous reviewer suggested and as required for this project, kindly endeavour to follow your chosen [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) on Figma, as strictly as possible. Aside from the fact that it looks better, this prepares you for a typical workplace experience where you may be required to reproduce a given design strictly.

Please, adjust the header on desktop version to match the design.

Your design:
![image](https://user-images.githubusercontent.com/20554583/184086959-f3db9863-a64b-4404-8834-6a316933ddc6.png)


Figma design:
![image](https://user-images.githubusercontent.com/20554583/184087259-bb4acc68-dbf9-4473-a3a9-d2c8009b73b2.png)